diff --git a/src/Silversolutions/Bundle/ToolsBundle/Services/EzHelperService.php b/src/Silversolutions/Bundle/ToolsBundle/Services/EzHelperService.php
index dbd1e85..e657a54 100755
--- a/src/Silversolutions/Bundle/ToolsBundle/Services/EzHelperService.php
+++ b/src/Silversolutions/Bundle/ToolsBundle/Services/EzHelperService.php
@@ -1040,7 +1040,7 @@ class EzHelperService
                     }
 
                     $query = new LocationQuery();
-                    $query->criterion = new Criterion\LogicalAnd($criteriaAnd);
+                    $query->filter = new Criterion\LogicalAnd($criteriaAnd);
 
                     /** @var \eZ\Publish\API\Repository\Values\Content\Search\SearchResult $searchResult */
                     $searchResult = $searchService->findLocations($query);
