diff --git a/src/Silversolutions/Bundle/EshopBundle/Services/Marketing/OlarkService.php b/src/Silversolutions/Bundle/EshopBundle/Services/Marketing/OlarkService.php
index f6aba8933..5806244dc 100644
--- a/src/Silversolutions/Bundle/EshopBundle/Services/Marketing/OlarkService.php
+++ b/src/Silversolutions/Bundle/EshopBundle/Services/Marketing/OlarkService.php
@@ -23,12 +23,12 @@ use Silversolutions\Bundle\EshopBundle\Entity\Basket;
 use Silversolutions\Bundle\EshopBundle\Product\ProductNode;
 
 use Siso\Bundle\ToolsBundle\Service\TemplateResolverServiceInterface;
-use Symfony\Bundle\TwigBundle\TwigEngine;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 
 class OlarkService implements TrackerInterface
 {
     /**
-     * @var TwigEngine $templating
+     * @var EngineInterface $templating
      */
     protected $templating;
 
@@ -45,12 +45,12 @@ class OlarkService implements TrackerInterface
 
     /**
      * dependency injection
-     * @param TwigEngine $templating
+     * @param EngineInterface $templating
      * @param TemplateResolverServiceInterface $templateResolver
      * @param ConfigResolverInterface $configResolver
      */
     public function __construct(
-        TwigEngine $templating,
+        EngineInterface $templating,
         TemplateResolverServiceInterface $templateResolver,
         ConfigResolverInterface $configResolver
 
